apps/condo/domains/billing/schema/RegisterBillingReceiptsService.test.js

```diff
@@ -398,35 +401,84 @@ describe('RegisterBillingReceiptsService', () => {
             })
 
             test('BillingReceipts are created if they have different categories', async () => {
-
                 const [organization] = await createTestOrganization(admin)
                 const [integration] = await createTestBillingIntegration(admin)
                 const [billingContext] = await createTestBillingIntegrationOrganizationContext(admin, organization, integration)
 
+                // create categories
+                const services = generateServicesData()
+                const defaultCategory = await BillingCategory.getOne(admin, {
+                    name: 'billing.category.housing.name',
```

Comment:
You can use constants, as all BillingCategories has the same id (UUID) on all servers

Reply:
Good point, 10x